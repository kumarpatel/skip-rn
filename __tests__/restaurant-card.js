import React from "react";
import renderer from "react-test-renderer";
import RestaurantCard from "../components/restaurant-card";
import mockData from "./mockSearchResults.json";

it("Snapshot - RestaurantCard", () => {
  const tree = renderer
    .create(
      <RestaurantCard restaurant={mockData.Restaurants[0]} onPress={() => {}} />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
