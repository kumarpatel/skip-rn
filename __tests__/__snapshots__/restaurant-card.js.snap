// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot - RestaurantCard 1`] = `
<View
  accessible={true}
  focusable={true}
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
>
  <View
    style={
      Object {
        "alignItems": "flex-start",
        "backgroundColor": "#fff",
        "borderColor": "grey",
        "borderRadius": 5,
        "borderWidth": 2,
        "flex": 1,
        "justifyContent": "center",
        "marginLeft": 5,
        "marginRight": 5,
        "marginTop": 10,
        "padding": 10,
      }
    }
  >
    <Image
      source={
        Object {
          "uri": "http://d30v2pzvrfyzpo.cloudfront.net/uk/images/restaurants/80086.gif",
        }
      }
      style={
        Object {
          "height": 70,
          "width": 70,
        }
      }
    />
    <View
      style={
        Object {
          "display": "flex",
          "flexDirection": "column",
        }
      }
    >
      <View
        style={
          Object {
            "display": "flex",
            "flexDirection": "row",
          }
        }
      >
        <Text>
          Name: 
          La Venice Pizzeria (Wood Fired)
        </Text>
      </View>
      <View
        style={
          Object {
            "display": "flex",
            "flexDirection": "row",
          }
        }
      >
        <Text>
          Rating: 
          5.08
        </Text>
      </View>
      <View
        style={
          Object {
            "display": "flex",
            "flexDirection": "row",
          }
        }
      >
        <Text>
          Cuisine Types:
           
          Italian | Pizza
        </Text>
      </View>
    </View>
  </View>
</View>
`;
